.github/workflows/continuous-integration.yml

```diff
@@ -73,6 +73,11 @@ jobs:
           ini-values: "zend.assertions=1"
           extensions: "pdo_sqlite"
 
+      - name: Install dependencies
+        run: |
+          composer update
```

Comment:
this should not be done here.

Instead, you need to move the command running `composer bin *` later in the workflow, after the step `Install dependencies with Composer`

Reply:
Composer usually complains that composer.lock is missing and performs update internally

Reply:
and that's the whole issue: you perform the install or update before the steps that modify the composer.json to change what will get installed.